packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -342,12 +352,16 @@ class DatePicker extends React.Component {
     );
 
     return (
-      <div className={styles['date-picker']}>
-        <ResponsiveElement
-          responsiveTo="window"
-          defaultElement={portalPicker}
-          small={popupPicker}
-        />
+      <div className={cx(['container-div'])}>
+        {required && <span className={cx(['required'])}>*</span>}
+        <div className={cx(['date-picker'])}>
+          <ResponsiveElement
+            className={cx(['date-picker'])}
```

Comment:
It looks like date-picker is added as a class twice now.

Reply:
Ooops. fixed it here: https://github.com/cerner/terra-framework/pull/504/commits/8035961724c3db36d70929ae027084bd24b5a293